ShelterModule/Models/Announcements/AnnouncementPutRequest.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace ShelterModule.Models.Announcements;
+
+public sealed class AnnouncementPutRequest
+{
+    public string? Title { get; init; } = null!;
```

Comment:
We don't need to explicitly set `Title` to `null`, because it's already a default value for nullable types. And even if we would like to do so for some reason, we don't need to add `!` because we don't need to lie to the compiler that the `null` we assign to this variable is not `null` (because it's legal to assign `null` to a nullable type).